buildkite/benchmark/run.py

```diff
@@ -692,6 +693,13 @@ def run_all_benchmark_groups(self):
         self.print_results()
 
         if len(self.failed_benchmark_groups()) > 0:
+            # Post a quick, dirty message to Slack
+            failure_names = "\n".join(bg.name for bg in self.failed_benchmark_groups())
+            Slack().post_message(
+                f"Failed Arrow benchmarks:\n{failure_names}\n\n"
+                f"Build log: {os.getenv('BUILDKITE_BUILD_URL')}"
+            )
```

Comment:
Since each machine seems to do env secrets differently, I'm going to move this check to the `schedule-and-publish` pipeline.